backend/src/test/java/ai/verta/modeldb/TestsInit.java

```diff
@@ -96,22 +101,25 @@ protected static void init() throws Exception {
 
     testConfig = TestConfig.getInstance();
     handleExecutor = FutureGrpc.initializeExecutor(testConfig.getGrpcServer().getThreadCount());
-    // Initialize services that we depend on
-    services = ServiceSet.fromConfig(testConfig, testConfig.artifactStoreConfig);
+
+    // TODO: FIXME: fix init flow as per spring bean initialization
```

Comment:
Is this a future task?  JIRA instead of to do in code?

Reply:
this is just a re-do of Anand's original PR; I only modified the one line that I called out.

Reply:
@ad-47 do we need a Jira for this TODO?